src/components/form/widgets/date-picker.ts

```diff
@@ -17,7 +17,8 @@ export class DatePicker extends React.Component {
 
     public render() {
         const props: WidgetProps = this.props;
-        const type = getDateType(props.schema);
+        this.props.schema.format = getDateFormat(props.schema);
```

Comment:
I don't think we should change this here :thinking: The consumer has given us a schema and it will probably be confusing and might lead to bugs if we change it.
We should either fix this issue in the backend when we serialize the schema or in the webclient when we add the property controls
I think I prefer the backend solution :slightly_smiling_face:

Reply:
Ah that is true, I will try to do it in the backend then 😄